ext/windows/run_facter_interactive.bat

```diff
@@ -0,0 +1,7 @@
+@echo Running facter on demand ...
+if exist "%~dp0environment.bat" (
+  call "%~dp0environment.bat" %0 %*
+) else (
+  SET PATH=%~dp0;%PATH%
+)
+elevate.exe "%~dp0facter_interactive.bat"

```

Comment:
Do we know the path to `elevate.exe` relative to the directory where `run_facter_interactive.bat` is installed?